cai_causal_graph/time_series_causal_graph.py

```diff
@@ -944,7 +946,7 @@ def from_adjacency_matrices(
                     time_delta_to_index[0] + (n_time_delta * column),
                 ] = 1
 
-        graph = cls.from_adjacency_matrix(adjacency_matrix_full, node_names)  # type: ignore
+        graph = cls.from_adjacency_matrix(adjacency_matrix_full, node_names, validate=validate)  # type: ignore
```

Comment:
Not sure how well the `validate=False` and the `construct_minimal=True` default will play when there is a cycle. I think `get_minimal_graph` will immediately fail.